packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -167,5 +167,5 @@ function toPerformanceEntryRepresentation(entry: RumPerformanceEntry): Performan
  * @returns number between 0 and 1 which represents tracing sample rate
  */
 function getRulePsr(configuration: RumConfiguration) {
-  return configuration.tracingSampleRate / 100
+  return typeof configuration.tracingSampleRate === 'number' ? configuration.tracingSampleRate / 100 : undefined
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏actually, we have a `isNumber` helper in `core/src/tools/utils`